src/myList.js

```diff
@@ -1,3 +1,5 @@
+/* The MyList class is a blueprint for creating objects that have a description, completed, and id
+property */
 class MyList {
   constructor(description, completed, id) {
     this.description = description;
```

Comment:
- If you use class constructor in one file you should use classes from each file.